### PR TITLE
Update actix-web depedency to 4.0.0

### PR DESCRIPTION
**Pull request by robjtede**

### CODE REVIEW

File: maud/src/lib.rs [hunk 1]
```diff
@@ -16,6 +16,9 @@ use core::fmt::{self, Arguments, Write};
 
 pub use maud_macros::{html, html_debug};
 
+#[cfg(feature = "actix-web")]
+extern crate actix_web_dep as actix_web;
+
```
Comment (lambda-fairy) [inline maud/src/lib.rs:21]: Can we revert this?

I'd rather keep the Actix-specific code in one place.

**Review by lambda-fairy — COMMENTED** [review]
Thanks @robjtede!

Look forward to when 4.0 is released :smile:

**Comment by lsr0** [comment]
Just a note: tested this PR via the source branch [1] and works nicely.

[1] `maud = { git = "https://github.com/robjtede/maud", branch = "aw4-rc", features = ["actix-web"] }`

**Comment by lambda-fairy** [comment]
Thank you. And congrats on the milestone!

**Comment by lsr0** [comment]
Hey, any chance of a maud release with actix 4.0 support?

**Pull request merged**